pr2eus/robot-interface.l

```diff
@@ -455,6 +455,10 @@
                    (dolist (limb target-limbs)
                      (setq ec-prev (elt end-coords-prev (position limb limbs))
                            ec-current (elt end-coords-current (position limb limbs)))
+                     ;; set midpoint of av as initial pose of IK
```

Comment:
I think you have to set this before `(dolist (limb target-limb)`, Please check with dual arm case.
https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/273/commits/19019cba7aaaf4a26a5f3efdc396610b1bf0c887#diff-15f6920b924513f6027d36b361f85bbdR455